test/src/org/exist/xquery/functions/xquery3/DeclareVariableTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  eXist Open Source Native XML Database
+ *  Copyright (C) 2001-2017 The eXist Project
+ *  http://exist-db.org
+ *
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public License
+ *  as published by the Free Software Foundation; either version 2
+ *  of the License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.exist.xquery.functions.xquery3;
+
+import static org.junit.Assert.assertEquals;
+
+import org.exist.test.EmbeddedExistTester;
+import org.junit.Test;
+import org.xmldb.api.base.ResourceSet;
+import org.xmldb.api.base.XMLDBException;
+
+public class DeclareVariableTest extends EmbeddedExistTester {
```

Comment:
Please don't use `EmbeddedExistTester` it will be removed here https://github.com/eXist-db/exist/pull/1251, please instead use the appropriate `ClassRule`

Also, I am wondering does this need to be added to `junit.xml` so it is executed?